adam-apis/src/test/scala/org/bdgenomics/adam/apis/java/JavaADAMContextSuite.scala

```diff
@@ -26,19 +26,73 @@ import org.bdgenomics.formats.avro.AlignmentRecord
 
 class JavaADAMContextSuite extends ADAMFunSuite {
 
-  sparkTest("can read a small .SAM file") {
+  sparkTest("can read and write a small .SAM file") {
     val path = copyResource("small.sam")
-    val ctx = new JavaADAMContext(sc)
-    val reads = ctx.loadAlignments(path)
-    assert(reads.jrdd.count() === 20)
+    val aRdd = sc.loadAlignments(path)
+    assert(aRdd.jrdd.count() === 20)
+
+    val newRdd = JavaADAMReadConduit.conduit(aRdd, sc)
```

Comment:
I would prefer including the code from `JavaADAMReadConduit` inline here, and similar for the other types.  That *Conduit classes are separate and public leads one to think they are more useful than they are.